packages/saber/lib/cli.js

```diff
@@ -54,6 +60,101 @@ cli
       .catch(handleError)
   })
 
+cli
+  .command(
+    'eject [app]',
+    `Copy the currently used theme's source code to a local folder.`
+  )
+  .option(
+    '--git',
+    'Pull code from Git, instead of node_modules, and add the theme as a submodule',
+    { default: false }
+  )
+  .option('--path <path>', 'Ejected theme destination', { default: './theme' })
+  .action(async (cwd = '.', options) => {
+    cwd = path.join(process.cwd(), cwd)
+    const { git } = options
+
+    const config =
+      configLoader.load({ cwd, files: configLoader.CONFIG_FILES }).data || {}
+    if (!config.theme) {
+      log.error('No theme specified in config.')
+      process.exit(1)
+    }
+
+    const destPath = path.join(cwd, options.path)
+    const relativeDest = path.relative(cwd, destPath)
+    if (await fs.pathExists(destPath)) {
+      log.error(
+        `The path ${
+          options.path
+        } already exists. Please specify a different one using "--path".`
+      )
+      process.exit(1)
+    }
+
+    const themePath = resolvePackage(config.theme, {
+      prefix: 'saber-theme-',
+      cwd
+    })
+    if (!themePath) {
+      log.error(
+        `Theme "${config.theme}" could not be found in your node_modules.`
+      )
+      process.exit(1)
+    }
+
+    if (git) {
+      const themePackage = configLoader.load({
+        cwd: themePath,
+        files: ['package.json']
+      }).data
+
+      if (themePackage.repository && themePackage.repository.url) {
+        const repo = themePackage.repository.url
+        await new Promise(resolve =>
+          exec(
+            `git submodule add "${repo}" "${relativeDest}"`,
+            { cwd },
+            (error, stdout, stderr) => {
+              if (error) {
+                log.error(`Could not add submodule: ${stderr.toString()}`)
+                process.exit(1)
+              }
+
+              if (options.verbose) {
+                log.verbose(stdout)
+                if (stderr) log.verbose(stderr)
+              }
+
+              log.info('Added theme as submodule.')
+              resolve()
+            }
+          )
+        )
+      } else {
+        log.error(
+          'The theme has no git repository specified within its package.json.'
+        )
+        process.exit(1)
+      }
+    } else {
+      try {
+        await fs.copy(themePath, destPath, {
+          filter: src => !src.endsWith('/node_modules')
+        })
+
+        log.info('Copied theme from node_modules.')
+      } catch (error) {
+        log.error(`Could not copy files: ${error.toString()}`)
+      }
+    }
+
+    log.info(
+      `Plese change "theme" in your Saber config to "./${relativeDest}".`
+    )
```

Comment:
We should probably remove the old theme from `package.json`?

And reinstall dependencies in the ejected directory?